CMakeLists.txt

```diff
@@ -598,6 +598,15 @@ endif()
 target_link_libraries(tvm PRIVATE ${TVM_LINKER_LIBS} ${TVM_RUNTIME_LINKER_LIBS})
 target_link_libraries(tvm_runtime PRIVATE ${TVM_RUNTIME_LINKER_LIBS})
 
+if(BUILD_FOR_HEXAGON)
+  include(FetchContent)
+  FetchContent_Declare(googletest SOURCE_DIR "${USE_HEXAGON_SDK}/utils/googletest/gtest")
+  set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+  FetchContent_MakeAvailable(googletest)
+  target_link_libraries(tvm_runtime PUBLIC gtest)
```

Comment:
Thinking we maybe should conditionally do this if a cmake variable is set so that we are only linking in gtest to the tvm_runtime when we want to do testing in order to avoid tvm_runtime bloat.

Reply:
Made USE_HEXAGON_GTEST an optional cmake param and enabling / disabling Hexagon gtest feature if that cmake param is specified with a path that exists.